util/disk/tempDir.go

```diff
@@ -77,17 +77,27 @@ func InitializeTempDir() error {
 		return err
 	}
 
+	// Create dir for MemoryUsageAlarmRecord.
+	_, err = os.Stat(filepath.Join(tempDir, "record"))
```

Comment:
For the dir name "record", we may create a constant variable for it.

Reply:
Addressed.